releases/net471/KnownIssues/517815-BCL Stack traces are missing source information for frames with debug information in the Portable PDB format.md

```diff
@@ -0,0 +1,19 @@
+# Stack traces are missing source information for frames with debug information in the Portable PDB format when running on .Net Framework 4.7.1
+
+## Symptoms
+
+An application that formats stack traces is missing source information for some or all frames. This includes stack traces formatted via [System.Exception.ToString()](https://docs.microsoft.com/en-us/dotnet/api/system.exception.tostring?view=netframework-4.7.1), [System.Exception.StackTrace](https://docs.microsoft.com/en-us/dotnet/api/system.exception.stacktrace?view=netframework-4.7.1) and [System.Diagnostics.StackTrace.ToString()](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stacktrace.tostring?view=netframework-4.7.1).
+The frames missing source information must reside in assemblies that have pdbs in the [Portable PDB format](https://github.com/dotnet/core/blob/master/Documentation/diagnostics/portable_pdb.md) present on disk.
```

Comment:
Since you're describing symptoms here, "must" is not necessary.

Reply:
My goal here was to qualify when you might be observing this issue vs. some other issue. For example you could build assemblies with any format pdb and then not put pdb on disk and you would observe the same missing source lines. However in that case you aren't hitting this issue, you are just failing to deploy your PDBs. I wanted to help people avoid some likely misdiagnosis.

Do you still I think I should remove 'must' or maybe phrase this a different way? I considered phrasing it as 'you observe _symptom_ when _condition_ is true' but it made a very long sentence whereas "must" helped me break the sentence apart. "You observe _symptom_. _condition_ must be met."

Reply:
Basically, symptoms defines the two conditions that must be present for this issue to apply. So in that sense, the "must" is not necessary. So the intent is already clear. I think the sentence reads much better with it removed. But I'll leave it up to you to decide, @noahfalk.